packages/api/tests/endpoints/token.js

```diff
@@ -214,19 +198,20 @@ test.serial('GET /token with a code and state results in a redirect containing t
 
   const locationHeader = new URL(response.headers.Location);
 
-  t.is(locationHeader.searchParams.get('token'), 'my-access-token');
+  t.is(locationHeader.searchParams.get('token'), jwtToken);
 });
 
-test.serial('When using Earthdata Login, GET /token with a code updates the user in DynamoDb', async (t) => {
-  const userName = randomString();
-  const userBefore = fakeUserFactory({ userName });
-  await userModel.create(userBefore);
+test.serial('When using Earthdata Login, GET /token with a code stores the access token in DynamoDb', async (t) => {
```

Comment:
It's probably worth reviewing the changes in this file and whether they make sense and whether the tests seem sufficient